### PR TITLE
fix(txsim): resolves discrepancies between suggested and implemented flags 

### PR DESCRIPTION
**Pull request by staheri14**
## Overview

During the interaction with txsim, I noticed certain inconsistencies between the suggested flags and the actual flags available in the implementation. The purpose of this PR is to reconcile the two aspects and ensure alignment.

Partially covers https://github.com/celestiaorg/celestia-app/issues/2383. 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: test/cmd/txsim/cli.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ defined sequences; recursive patterns between one or more accounts which will co
 transactions. You can use flags or environment variables (TXSIM_RPC, TXSIM_GRPC, TXSIM_SEED, 
 TXSIM_POLL, TXSIM_KEYPATH) to configure the client. The keyring provided should have at least one
 well funded account that can act as the master account. The command runs until all sequences error.`,
-		Example: "txsim --key-path /path/to/keyring --rpc-endpoints localhost:26657 --grpc-endpoints localhost:9090 --seed 1234 --poll-time 1s --blob 5",
```
Comment (staheri14) [inline test/cmd/txsim/cli.go:66]: [To reviewers] txsim does not support `rpc--endpoints` option. 
Also, the `grpc-endpoints` flag should be singular.

Comment (rootulp) [reply test/cmd/txsim/cli.go:66]: good catch! Previously it was `rpc-endpoints` and `grpc-endpoints`. Now it's `grpc-endpoint`.  See this recent change: https://github.com/celestiaorg/celestia-app/pull/2326

**Review by rootulp — APPROVED** [review]

**Comment by rootulp** [comment]
Likely closes https://github.com/celestiaorg/celestia-app/issues/2383

**Comment by staheri14** [comment]
> Likely closes #2383

Thanks, Yes, likely it does. However, it also mentions some issues with the lack of new lines in help messages which I haven't encountered (and cannot see any issue). So, going to just link it in the PR description without the `close` verb.

**Pull request merged**